cluster/gce/gci/configure.sh

```diff
@@ -376,6 +378,10 @@ function install-node-problem-detector {
 function install-cni-network {
   mkdir -p /etc/cni/net.d
   case "${NETWORK_POLICY_PROVIDER:-flannel}" in
+    mizar)
+    setup-mizar-cni-conf
+    install-mizar-cni-bin
+    ;;
```

Comment:
From: https://github.com/CentaurusInfra/mizar/blob/dev-next/docs/user/getting_started.md, Kernel version need be at least 5.6-rc2 on every node, should we add kernelupdate.sh to Mizar installation?

Reply:
Yes. I prefer that we don't go that route. We should instead try to fix kube-up for 20.04 which has the latest kernel